marimo/_plugins/ui/_impl/tables/pandas_table.py

```diff
@@ -25,15 +29,33 @@ def create() -> type[TableManager[Any]]:
         class PandasTableManager(TableManager[pd.DataFrame]):
             type = "pandas"
 
-            def to_csv(self) -> bytes:
+            def to_csv(
+                self, format_mapping: Optional[FormatMapping] = None
+            ) -> bytes:
                 has_headers = len(self.get_row_headers()) > 0
-                return self.data.to_csv(
+                _data = self.data.copy()
```

Comment:
this might be an unnecessary copy, you could just do `_data = self.data`

Reply:
you could always write a test too to make sure the incoming df does not get modified